components/src/core/list-item-tag/list-item-tag.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'pxb-list-item-tag',
+  templateUrl: './list-item-tag.component.html',
+  styleUrls: ['./list-item-tag.component.scss'],
+})
+
+export class ListItemTagComponent {
+
+  ngOnInit() {
+
+    console.log( "LOOK HERE " + this.fontColor);
+  }
+
+  @Input() label: string;
```

Comment:
@Inputs should be placed before the ngOnInit.

Reply:
Thanks for the tip.